### DISCUSSION TITLE
Moving/Copying Improvements

### DISCUSSION THREAD

**Discussion by CEbbinghaus** [filebrowser/filebrowser#1767] [Ideas]
I am using filebrowser mainly to move files between folders and different spots and it's Incredibly difficult to use the User interface to copy as well as not being able to see the progress of the copy/move so it's hard to tell when it's completed. A page with all current transfers and their progress as well as the ability to cancel them would make the moving of files much more usable.

**Comment by goni05** [comment]
Even just an acknowledgement when clicking on the button by closing the Copy/Move option to allow continued use of the front end would be nice.  It's not clear when I click the button that it has actually occurred.

**Comment by cybrdyne** [comment]
Agreed, acknowledgement, progress status, and queue list is major feature that is missing!
